lib/trino-collect/src/test/java/io/trino/collect/cache/TestEvictableCache.java

```diff
@@ -75,17 +75,24 @@ public void testLoad()
     public void testEvictBySize()
             throws Exception
     {
+        int maximumSize = 10;
         Cache<Integer, Integer> cache = EvictableCacheBuilder.newBuilder()
-                .maximumSize(10)
+                .maximumSize(maximumSize)
                 .build();
 
         for (int i = 0; i < 10_000; i++) {
             int value = i * 10;
             assertEquals((Object) cache.get(i, () -> value), value);
         }
         cache.cleanUp();
-        assertEquals(cache.size(), 10);
-        assertEquals(((EvictableCache<?, ?>) cache).tokensCount(), 10);
+        assertEquals(cache.size(), maximumSize);
+        assertEquals(((EvictableCache<?, ?>) cache).tokensCount(), maximumSize);
+
+        // Ensure cache is effective, i.e. some entries preserved
+        int lastKey = 10_000 - 1;
```

Comment:
nit: extract 10_000 to variable

Reply:
i chose not to apply this one, hope it's fine